ResultTests/ResultTests.swift

```diff
@@ -2,11 +2,11 @@
 
 final class ResultTests: XCTestCase {
 	func testMapTransformsSuccesses() {
-		XCTAssertEqual(success.map(count) ?? 0, 7)
+		XCTAssertEqual(success.map { $0.characters.count } ?? 0, 7)
 	}
 
 	func testMapRewrapsFailures() {
-		XCTAssertEqual(failure.map(count) ?? 0, 0)
+		XCTAssertEqual(failure.map { $0.characters.count } ?? 0, 0)
```

Comment:
Yea, but I actually like that they removed `CollectionType` from string for the sake of explicitness.